docs/product-installation-guide/src/main/asciidoc/clustering-runtime-managed-proc.adoc

```diff
@@ -8,7 +8,7 @@
 . Set up a {EAP} 7.1 cluster as described in  https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/configuring_a_red_hat_jboss_eap_7_cluster/[_Configuring a {EAP} 7 Cluster_].
```

Comment:
This is a huge document. AFAIK, clustering the runtime environment only requires:
# shared DB settings
# clustered EJB timer implementation
# kie container alignment

So asking customers to follow this long document just to meet these two requirements is quite an overhead IMO. Moreover, I'm not even sure if this results in clustered EJB timers (it is not explicitly stated). @maciejs has the final word, but I'd think that the whole clustering section could be replaced.

Reply:
No idea on why you mentioned me here.

Reply:
@maciejs I think @martinweiler meant to mention me instead, anyway thanks for replying

@martinweiler there might be many aspects of the EAP that is good to be taken into consideration for cluster setup and thus it makes sense to refer to official cluster guide. One of such things is the JMS provider that makes sense to have clustered.